src/backend/app/projects/project_routes.py

```diff
@@ -876,16 +877,30 @@ async def update_project_form(
     )
 
     sql = """
-        INSERT INTO projects
-            (xlsform_content)
-        VALUES
-            (%(xls_data)s)
+        UPDATE projects
+        SET
+            xlsform_content = %(xls_data)s,
+            author_id = %(author_id)s,
```

Comment:
The author_id is the original author of the project. Should this be updated too when the form is updated? Was it causing an error without it?

Reply:
Good point, there was an error initially due to insert command and I had to put not null constraints values too ( `author_id` ). For update we don't need it actually, you are right :+1: